### PR TITLE
fix(stable base nodes): minor updates and move to stable

### PR DESCRIPTION
**Pull request by brianp**
Description
---
Added a little extra info regarding pruned nodes and peer validation.
Moved RFC to stable.

Motivation and Context
---
Gett'n things updated.

### CODE REVIEW

File: src/RFC-0110_BaseNodes.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ Tari Base Nodes MUST carry out the following tasks:
 * provide historical block information to peers that are syncing.
 
 Once the Digital Assets Network (DAN) goes live, Base Nodes will also need to support the tasks described in
-[RFC-0300_DAN](RFCD-0300_DAN.md). These requirements are omitted for the moment.
+[RFC-0300_DAN](RFCD-0300_DAN.md). These requirements may involve but are not limited to:
```
Comment (CjS77) [inline src/RFC-0110_BaseNodes.md:81]: RFC-0304

**Review by CjS77 — COMMENTED** [review]
LG, but RFC link needs to be updated

**Comment by CjS77** [comment]
Points/inaccuracies that should be addressed and/or confirmed before merging:

* Cut-through has been deacivated in Tari
* Update "Block/Transaction Weight" section language to be authoritative rather than the conditional form (and confirm that the values match the code).
* BroadcastStrategy might be an outdated reference.

**Comment by brianp** [comment]
> Points/inaccuracies that should be addressed and/or confirmed before merging:

- ✅ Cut-through has been deactivated in Tari
Confirmed as per RFC-0201_TariScript cut-through has been deactivated in Tari to prevent a bad actor from utilizing cut through to spend a UTXO that they know the blinding factor too, but Tari script is being used enforce a particular public key.
- ✅ Update "Block/Transaction Weight" section language to be authoritative rather than the conditional form (and confirm that the values match the code).
Values were already confirmed, language updated, table formatting fixed.
- ✅ BroadcastStrategy might be an outdated reference.
Still seems relevant but updated the language to be more specific about which strategy as multiple are mentioned in the glossary.

**Review by stringhandler — APPROVED** [review]

**Pull request merged**